src/i18n/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_program(GETTEXT_MSGFMT_EXECUTABLE msgfmt)
+if(NOT GETTEXT_MSGFMT_EXECUTABLE)
+    message(FATAL_ERROR "msgfmt not found")
+endif()
+
+find_program(GETTEXT_MSGMERGE_EXECUTABLE msgmerge)
+if(NOT GETTEXT_MSGMERGE_EXECUTABLE)
+    message(FATAL_ERROR "msgmerge not found")
+endif()
+
+find_program(GETTEXT_MSGATTRIB_EXECUTABLE msgattrib)
+if(NOT GETTEXT_MSGATTRIB_EXECUTABLE)
+    message(FATAL_ERROR "msgattrib not found")
+endif()
+
+SET(TRANSLATIONS
+    de
+    en
+    es
+    it
+    ru
+)
+
+foreach(_trans ${TRANSLATIONS})
+add_custom_command(
+   COMMENT "Generate Locale '${_trans}'"
+   OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po"
+   POST_BUILD
+   COMMAND mkdir -p Locale/${_trans}/LC_MESSAGES
+   COMMAND ${GETTEXT_MSGATTRIB_EXECUTABLE} --translated "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po" -o "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po"
+   COMMAND ${GETTEXT_MSGMERGE_EXECUTABLE} --update --no-fuzzy-matching "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po" "${CMAKE_CURRENT_SOURCE_DIR}/Locale/en.po"
```

Comment:
`Line 31` uses `--update` ... uhm whatever .. I have no idea

Yeah `Line 32` just makes sure the empty messages are in the po file if there isn't any en-translation.. shouldn't be needed if en is up to date always.. it got generated for the translators..

Anyways.. whatever .. do what you think is best because I have no idea how to work with translations.

You also can change the `*.ui` files to directly use english (instead of keys)
But I have no idea how to update translation in english without destroying other translation that reference to these english translations (as `msgid`)...

The way it's now, **it worked**-tm. I am not proud of this translation-mess

It just had 2 things todo
- make sure all new `msgid` are in (from \*.ui and \*.cpp/\*.h)
- make sure empty/missing translation have en-translation

`msgcat` and `msgmerge` behaved very different from what I imagined and this is the crazy **get it somehow working**-solution.